chat-plugins/hangman.js

```diff
@@ -225,7 +225,7 @@ exports.commands = {
 
 		guess: function (target, room, user) {
 			if (!target) return this.parse('/help guess');
-			if (!room.game || room.game.gameType !== 'hangman') return this.errorReply("There is no game of hangman running in this room.");
+			if (!room.game || room.game.title !== 'Hangman') return this.errorReply("There is no game of hangman running in this room.");
```

Comment:
`.title` is for display, and is not guaranteed to be or not to be anything.

We might want to preserve the concept of `.gameType`... Although maybe call it something like `.gameid`